examples/cloud-controller-manager/persistent-volume-label-initializer-config.yaml

```diff
@@ -0,0 +1,13 @@
+kind: InitializerConfiguration
+apiVersion: admissionregistration.k8s.io/v1alpha1
+metadata:
+  name: pvlabel-initconfig
```

Comment:
does this name allow dots?  `pvlabel.kubernetes.io` would be fully qualified, rigth?